mailbox/elasticsearch/src/main/java/org/apache/james/mailbox/elasticsearch/events/ElasticSearchListeningMessageSearchIndex.java

```diff
@@ -109,13 +109,24 @@ public ListenerType getType() {
     @Override
     public void add(MailboxSession session, Mailbox mailbox, MailboxMessage message) throws MailboxException {
         try {
+            LOGGER.info("Indexing mailbox {}-{} of user {} on message {}",
+                    mailbox.getName(), mailbox.getMailboxId(),
+                    session.getUser().getUserName(),
+                    message.getUid());
             indexer.indexMessage(indexIdFor(mailbox, message.getUid()), messageToElasticSearchJson.convertToJson(message, ImmutableList.of(session.getUser())));
         } catch (Exception e) {
             try {
-                LOGGER.warn("indexing message {} without attachments ", message.getUid());
+                LOGGER.warn("Indexing mailbox {}-{} of user {} on message {} without attachments ",
```

Comment:
Place holders "{}"mess up when mixed with exceptions.

(The stack trace is not displayed)

So please back up here to String concatenation, or using an other mechanism for interpolation.